zerver/lib/mention.py

```diff
@@ -17,8 +17,8 @@
 USER_GROUP_MENTIONS_RE = re.compile(
     rf"{BEFORE_MENTION_ALLOWED_REGEX}@(?P<silent>_?)(\*(?P<match>[^\*]+)\*)"
 )
-
-wildcards = ["all", "everyone", "stream"]
+topic_wildcards = ["topic"]
+wildcards = ["all", "everyone", "stream"] + topic_wildcards
```

Comment:
Since we never intend to mutate `topic_wildcards` or `wildcards`, do you think it is better to use `frozenset`s here? This can be done in a separate refactoring commit.

Reply:
Yeah, a frozenset sounds correct to me.